src/util.h

```diff
@@ -69,15 +70,50 @@ inline std::string _(const char* psz)
 void SetupEnvironment();
 bool SetupNetworking();
 
+enum LogFlags : uint32_t {
+    LOG_NONE        = 0,
+    LOG_NET         = (1 <<  0),
```

Comment:
Another approach would be wrapping it in a C++ namespace (for example:
```c++
namespace bclog {
enum flags {
  NET,
  MEMPOOL,
  ...
};
}
```

at which point you can use `bclog::NET | bclog::MEMPOOL`.